frame/executive/src/lib.rs

```diff
@@ -249,7 +249,7 @@ where
 			frame_system::InitKind::Full,
 		);
```

Comment:
Do we leave `initialize` without weight on purpose?

Reply:
The entire block initialization is included in the "block exection weight", so:

```
<System::BlockExecutionWeight as frame_support::traits::Get<_>>::get())